pkgs/development/misc/h3/default.nix

```diff
@@ -36,6 +36,8 @@ let
         (lib.cmakeBool "ENABLE_FORMAT" false)
         (lib.cmakeBool "ENABLE_LINTING" false)
         (lib.cmakeBool "BUILD_FILTERS" withFilters)
+        # Fix the build with CMake 4.
+        "-DCMAKE_POLICY_VERSION_MINIMUM=3.5"
```

Comment:
Actually, I take back my review. `h3_4` is building successfully, it's only `h3_3` that's failing.

Perhaps gate this behind a version check, like I did in my (temporary) commit:
https://github.com/NixOS/nixpkgs/pull/423867/commits/e8d1806211eccefa18777b6138a1f678b0bbac52